kolibri/plugins/learn/assets/src/views/assessment-wrapper/exercise-attempts/index.vue

```diff
@@ -73,10 +73,13 @@
         if (this.waitingForAttempt) {
           xPos += ANSWER_WIDTH;
         }
+        if (this.isRtl) {
+          xPos *= -1;
+        }
         const style = {};
         // translateZ(0) is there to try and force GPU-acceleration.
         // (see e.g. http://blog.teamtreehouse.com/increase-your-sites-performance-with-hardware-accelerated-css)
-        style.transform = `translate(${xPos}px) translateZ(0)`;
```

Comment:
Removed the `translateZ(0)` because it was unnecessary since it never changed.

Reply:
see the comment directly above

Reply:
D'oh

Reply:
@indirectlylit  Added back in, but I doubt it makes any noticeable difference.

Reply:
True. just trying to save the kids a little battery